populus/chain.py

```diff
@@ -714,7 +714,10 @@ def __enter__(self, *args, **kwargs):
         self.stack.enter_context(self.geth)
 
         if self.geth.is_mining:
-            self.geth.wait_for_dag(600)
+            # On Amazon EC2 2 VCPU small unit you get 32% of DAG generated
+            # within 600 seconds. Estimate is
+            # we can generate whole DAG in 40 minutes.
+            self.geth.wait_for_dag(60*40)
```

Comment:
+1 to this.  I think it might still be useful to have an environment based configuration option for this since you may want to use one value when testing locally and a much higher value on your CI solution and in those cases it's nice to not have to account for that difference in your code.